### DISCUSSION TITLE
Use underscores for system fields

### DISCUSSION THREAD

**Discussion by ryanleichty** [craftcms/cms#15593] [Ideas] [answered]
Something I'm fan of in [Sanity](https://www.sanity.io/docs/naming-things) is that they use _underscores_ for system fields rather than reserving all of the best field names (something that Craft does which has always bothered me).

In Craft, field names like `title`, `url`, `status`, etc., are reserved words and can't be used as custom field names. 

In Sanity, they use names like `_title` or `_url` for system fields, allowing you to use `title` or `url` freely.

**Comment by brandonkelly** [comment, marked as answer]
This would be a massive breaking change for all existing projects, so not something we could seriously consider.
